Assets/Scripts/Player/PlayerControls.cs

```diff
@@ -243,7 +245,12 @@ public override void Update()
 			     player.stats.CanUseStat( Stat.Stamina ) &&
 			     player.physics.ClimbCheck() )
 			{
-				player.physics.ClimbSurface( player.controls.GetMoveInput() );
+				Vector3 moveInput = player.controls.GetMoveInput();
+				player.physics.ClimbSurface( moveInput );
+				player.animator.SetFloat( "moveSpeed", moveInput.magnitude );
+				player.animator.SetFloat( "verticalSpeed", moveInput.z );
+				player.animator.SetFloat( "horizontalSpeed", moveInput.x );
+				player.animator.SetFloat( "slope", Mathf.Abs( Vector3.Dot( moveInput, Vector3.forward ) ) );

```

Comment:
I use it to blend between horizontal and vertical climb animations. I'm pretty sure `Vector3.forward` is correct in this case because the result from `GetMoveInput()` is a formatted as `<horizontal input, 0.0f, vertical input>` and I want to get a value that goes between 0 and 1 depending on whether the joystick is left/right or up/down.